interface/wx/utils.h

```diff
@@ -928,24 +928,106 @@ wxString wxGetOsDescription();
     contain the following values:
-    @beginTable
-    @row3col{<b>Windows OS name</b>, <b>Major version</b>, <b>Minor version</b>}
-    @row3col{Windows 10,               10, 0}
-    @row3col{Windows Server 2016,      10, 0}
-    @row3col{Windows 8.1,               6, 3}
-    @row3col{Windows Server 2012 R2,    6, 3}
-    @row3col{Windows 8,                 6, 2}
-    @row3col{Windows Server 2012,       6, 2}
-    @row3col{Windows 7,                 6, 1}
-    @row3col{Windows Server 2008 R2,    6, 1}
-    @row3col{Windows Server 2008,       6, 0}
-    @row3col{Windows Vista,             6, 0}
-    @row3col{Windows Server 2003 R2,    5, 2}
-    @row3col{Windows Server 2003,       5, 2}
-    @row3col{Windows XP,                5, 1}
-    @endDefList
+    <table>
+        <tr>
+            <th>Windows OS name</th>
+            <th>Major version</th>
+            <th>Minor version</th>
+            <th>Build number</th>
+        </tr>
+        <tr>
+            <td>Windows 11</td>
+            <td>10</td>
+            <td>0</td>
+            <td>&gt;= 22000</td>
+        </tr>
+        <tr>
+            <td>Windows Server 2022</td>
+            <td>10</td>
+            <td>0</td>
+            <td>&gt;= 22000</td>
+        </tr>
+        <tr>
+            <td>Windows 10</td>
+            <td>10</td>
+            <td>0</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>Windows Server 2016</td>
+            <td>10</td>
+            <td>0</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>Windows 8.1</td>
+            <td>6</td>
+            <td>3</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>Windows Server 2012 R2</td>
+            <td>6</td>
+            <td>3</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>Windows 8</td>
+            <td>6</td>
+            <td>2</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>Windows Server 2012</td>
+            <td>6</td>
+            <td>2</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>Windows 7</td>
+            <td>6</td>
+            <td>1</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>Windows 2008 R2</td>
+            <td>6</td>
+            <td>1</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>Windows Vista</td>
+            <td>6</td>
+            <td>0</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>Windows Server 2008</td>
+            <td>6</td>
+            <td>0</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>Windows Server 2003 R2</td>
+            <td>5</td>
+            <td>2</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>Windows Server 2003</td>
+            <td>5</td>
+            <td>2</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>Windows Server XP</td>
```

Comment:
```suggestion
            <td>Windows XP</td>
```

Reply:
Thanks for noticing this copy and paste error